chains/scroll.json

```diff
@@ -0,0 +1,39 @@
+{
+  "alias": "scroll",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": true
+      },
+      "url": "https://api.scrollscan.com/api"
+    },
+    "browserUrl": "https://scrollscan.com/"
+  },
+  "id": "534352",
+  "name": "Scroll",
```

Comment:
Note to myself: See and implement [this](https://docs.scroll.io/en/developers/transaction-fees-on-scroll/) while calculating transaction fee. It looks a bit different than optimism stack.